### PR TITLE
feat: Add description of integration prefill feature

### PR DESCRIPTION
**Pull request by valekjo**

### CODE REVIEW

File: sources/platform/integrations/actors/integration_ready_actors.md [hunk 1]
```diff
@@ -69,9 +69,19 @@ It's also possible to combine both approaches, which is useful for development p
 
 ```js
 const { payload, datasetId } = await Actor.getInput();
-const datasetIdToProcess = payload?.resource?.defaultDatasetId ?? datasetId;
+const datasetIdToProcess = datasetId || payload?.resource?.defaultDatasetId;
 ```
 
+To make the integration process smoother it's possible to define an input that's going to be prefilled when actor is being used as an integration. You can do that in the "Settings" tab of your Actor, on the "Integrations" form. In our example, we'd use:
```
Comment (katacek) [inline sources/platform/integrations/actors/integration_ready_actors.md:75]: To make the integration process smoother, it's possible to define an input that's going to be prefilled when Actor

Suggestion: 
-> adding comma after smoother
-> actor -> Actor

**Review by katacek — APPROVED** [review]
Nice, just nitpicks (see code) but approving already 👌

File: sources/platform/integrations/actors/integration_ready_actors.md (hunk 1 above)
Comment (valekjo) [reply sources/platform/integrations/actors/integration_ready_actors.md]: Fixed

File: sources/platform/integrations/actors/integration_ready_actors.md [hunk 2]
```diff
@@ -69,9 +69,19 @@ It's also possible to combine both approaches, which is useful for development p
 
 ```js
 const { payload, datasetId } = await Actor.getInput();
-const datasetIdToProcess = payload?.resource?.defaultDatasetId ?? datasetId;
+const datasetIdToProcess = datasetId || payload?.resource?.defaultDatasetId;
 ```
 
+To make the integration process smoother, it's possible to define an input that's going to be prefilled when Actor is being used as an integration. You can do that in the "Settings" tab of your Actor, on the "Integrations" form. In our example, we'd use:
```
Comment (m-murasovs) [inline sources/platform/integrations/actors/integration_ready_actors.md:75]: Just adjustments to make it follow conventions
```suggestion
To make the integration process smoother, it's possible to define an input that's going to be prefilled when your Actor is being used as an integration. You can do that in the Actor's **Settings** tab, on the **Integrations** form. In our example, we'd use:
```

**Review by m-murasovs — APPROVED** [review]

**Pull request merged**